grammars/SeedPython.g4

```diff
@@ -1,39 +1,97 @@
-/**
- * Copyright 2021 The Aha001 Team.
+/*
```

Comment:
For the LICENCE notice: I think we are not re-using this 3rd-party code. Actually, we are modifying it into a derived version, with a different license (given that the two licenses, i.e., Apache and MIT, are compatible). It's common to declare both the license of the derived work and the license of the original work in the file header. One of the best practices is:

```
Copyright 2021 The Aha001 Team

... The Apache License notice ...

This is a derived work of https://github.com/bkiers/Python3-parser.
Here is the original license notice:

Copyright (c) 2014 by Bart Kiers

... The original MIT license notice ...
```

See https://softwareengineering.stackexchange.com/questions/157968/how-to-manage-a-copyright-notice-in-an-open-source-project for an open discussion.

And, the Apache guidelines suggest a NOTICE file to list all the 3rd-party dependencies and their copyright and permission notices. See https://infra.apache.org/licensing-howto.html#permissive-deps . We can consider creating this NOTICE file later, since we have already included their source code notice.